lms/djangoapps/verify_student/migrations/0012_populate_SSPV.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-01-11 11:32
+from __future__ import unicode_literals
+
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import migrations
+
+
+def populate_expiry_date(apps, schema_editor):
```

Comment:
There are around 700k records in this table and this will break the production. I believe we should move this into a management command and update the rows in batches.